src/utils/safe-env.js

```diff
@@ -0,0 +1,18 @@
+function ensureEnv(key, msg = `process.env.${key} is not defined`) {
```

Comment:
Not a change request but you may want to use [dotenv-safe](https://github.com/rolodato/dotenv-safe) in the future for this. 

Plays nicely with `env-cmd`, e.g. https://github.com/kleros/gtcr-event-mirror/blob/b540a257a49ba50114a90a036fdbd38874eb9254/package.json#L10

Reply:
I didn't know it could be used with `dotenv-safe`. Where did you learn this? I couldn't find it in the docs.